File: problem_eleven.js
// Find the number of zeros at the end of the factorial number
var number = 15
function fun(){
    var num = 1
    for(var i = 0;i < number;i++){
        var val = i*num
        num+=val
    }
    var arr = String(num).split("")
    var count = 0
    for(var i = arr.length -1; i > 0;i--){
        if (parseInt(arr[i])== 0){
            count+=1
        }
        if(parseInt(arr[i])!=0){
            break
        }
    }
    return count
}
console.log(fun(number))

File: problem_eleven.py
# Find the number of zeros at the end of the factorial number
number = int(input("Enter the number:"))
def Factorial(number):
    num = 1
    for i in range(number):
        var = i* num
        num += var
    count = 0
    arr = list(str(num))
    for i in range(len(arr)-1,0,-1):
        if int(arr[i])== 0:
            count+=1
        if int(arr[i])!= 0:
            break
    return count       
print(Factorial(number))    


